wfl/calculators/utils.py

```diff
@@ -147,6 +147,17 @@ def save_results(atoms, properties, results_prefix=None):
     if 'energies' in properties:
         atoms_results['energies'] = atoms.get_potential_energies()
 
+    if "extra_results" in dir(atoms.calc):
```

Comment:
What happened to the test confirming that `results_prefix` is not None when extra results are present?

Reply:
I initially misunderstood the logic, added that back in.